### ISSUE TITLE
As a security auditor, I want to be able to review vuln logs, so that I can ensure they are being addressed

### ISSUE THREAD

**Issue by hannahkane** [18F/fs-permit-platform#166]
## Acceptance Criteria
- [ ]

## Tasks
- [X] Configure snyk csv output
- [X] Debug snyk storage of artifacts
- [X] Implement so that pushes to s3.
- [X] Implement chron job so that it runs nightly
- [X] Copy to fs-permit-platform

## Definition of Done
- [X] Code unit / e2e tested and comply with QWASP
- [X] No new security vulnerabilities
- [ ] Internal team code review
- [ ] Documentation / readme.md / CircleCI updated

**Comment by lauraGgit** [comment]
Waiting on review of first implementation & push to s3, then will work on on chron addition + copy.

**Comment by lauraGgit** [comment]
Added `chron` pr, and now having review of this - with other changes.

**Comment by lauraGgit** [comment]
FS Permit Platform addressed by #170

**Comment by lauraGgit** [comment]
Merged into master on `fs-permit-platform`
